deploy/kubernetes/seqr/seqr.gcloud.yaml

```diff
@@ -167,8 +160,6 @@ spec:
           value: "{{ DATABASE_BACKUP_BUCKET }}"
         - name: RUN_CRON_JOBS
           value: "{{ RUN_CRON_JOBS }}"
-        - name: STATIC_MEDIA_DIR
-          value: "{{ STATIC_MEDIA_DIR }}"
```

Comment:
we should remove this variable from `shared-settings.yaml` since its no longer used

Reply:
will do 👍

Reply:
will handle this over in #2397